packages/mdc-textfield/index.js

```diff
@@ -309,22 +309,6 @@ class MDCTextField extends MDCComponent {
     this.foundation_.setHelperTextContent(content);
   }
 
-  /**
-   * Sets the aria label of the icon.
-   * @param {string} label
-   */
-  set iconAriaLabel(label) {
```

Comment:
Why were `set iconAriaLabel` and `set iconContent` methods removed?

Reply:
This makes me suspicious of a bad sync with master... this is clobbering #2771. Please check to make sure those changes are intact.

Reply:
Definitely a bad sync on my part! Added them back in.